admin-dev/themes/new-theme/js/components/modal/modal.ts

```diff
@@ -221,6 +221,8 @@ export class Modal implements ModalType {
 
   hide(): void {
     this.$modal.modal('hide');
+    // Sometimes modal animation is still in progress and hiding fails, so we attach event listener for that case.
+    this.$modal.on('shown.bs.modal', () => this.$modal.modal('hide'));
```

Comment:
ping @NeOMakinG had a discussion in private, this is what I ended up with, is it ok?

Reply:
```suggestion
    this.$modal.on('shown.bs.modal', () => {
      this.$modal.modal('hide');
      this.$modal.off('shown.bs.modal');
    });
```

What happens if you show the modal again for any other reason? It will auto hide it when it's shown, but maybe you don't want to